package-examples/ghost/Kptfile

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: ghost
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: sample description
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/set-namespace:v0.3.4
```

Comment:
Was curious we aren't using `set-namespace:v0.4.1` ?

Reply:
`v0.4.1` does not accept duplicate GVKNN which means using `v0.4.1` users need to specify the set-namespace in the children package rather than using it in the parent. 
FYI, I think that is an unnecessary requirement. So the latest set-namespace (unreleased) removes this requirement.